config/runtime/error_handler.go

```diff
@@ -13,6 +12,11 @@ import (
 	"github.com/avenga/couper/handler"
 )
 
+var wildcardMap = map[string][]string{
```

Comment:
Can we somehow create those mappings as follow-up generation task via `errors/generate` ?

Reply:
Good idea. However, here we only need mappings for errors related to `api`/`endpoint.`